serverlessworkflow/modules/ROOT/pages/data-index/data-index-usecase-singleton.adoc

```diff
@@ -0,0 +1,206 @@
+= Deploying Data Index and {product_name} application on Minikube
```

Comment:
Same comment related to the #518 PR.

Reply:
This comment is for the future, after merging this and refactor to incorporate Data Index on Operator guide.